### PR TITLE
Upgrade to OptaPlanner from 7.43 to 8.0.0.Final

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

File: optaplanner-quickstart/pom.xml [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
     <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-    <optaplanner-quarkus.version>7.43.0.Final</optaplanner-quarkus.version>
+    <optaplanner-quarkus.version>8.0.0.Final</optaplanner-quarkus.version>
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
     <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
     <docker-plugin.version>0.28.0</docker-plugin.version>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
```
Comment (ge0ffrey) [inline optaplanner-quickstart/pom.xml:19]: Is this acceptable?

OptaPlanner 8.0.0.Final.jar has java 11 compatible class files. It targets 11. It doesn't work on Java 8.

File: optaplanner-quickstart/pom.xml [hunk 2]
```diff
@@ -155,11 +169,9 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
-                  <systemPropertyVariables>
+                  <systemProperties>
```
Comment (gsmet) [inline optaplanner-quickstart/pom.xml:172]: Why did you change that? IIRC `systemProperties` is deprecated.

Comment (ge0ffrey) [reply optaplanner-quickstart/pom.xml:172]: Fixing :)

**Comment by ge0ffrey** [comment]
@gsmet Fixed + ideally this is in time for the 1.10 release.
OptaPlanner 8.0.0.Final is now compatible with Java 8 again and has been released. It should be on Maven Central any moment now.

File: optaplanner-quickstart/pom.xml (hunk 1 above)
Comment (ge0ffrey) [reply optaplanner-quickstart/pom.xml:19]: OptaPlanner 8.0.0.Final now works on Java 8 too.
 Is it fine that this quickstarts defaults to 11 to stop 11 from proliferating?

**Comment by gsmet** [comment]
Force pushed to get back to UTF-8. Let's see if CI is happy now.

**Comment by ge0ffrey** [comment]
Thanks @gsmet !
